packages/strapi-plugin-documentation/services/Documentation.js

```diff
@@ -195,7 +195,14 @@ module.exports = {
   },
 
   createDocObject: array => {
-    return array.reduce((acc, curr) => _.merge(acc, curr), {});
+    return _.sortBy(array, t => t.tags[0] && t.tags[0].name).reduce((acc, curr) => {
```

Comment:
I feel like we should just update the areObjectsEquals to use lodash deepEqual instead. this would make things a lot more reliable.